contexts/citizenship-v1.jsonld

```diff
@@ -0,0 +1,24 @@
+{
+  "@context": {
+    "@version": 1.1,
+    "id": "@id",
+    "type": "@type",
+    "ctzn": "https://w3id.org/citizenship#",
+    "schema": "http://schema.org/",
+    "xsd": "http://www.w3.org/2001/XMLSchema#",
+
+    "PermanentResidentCard": "ctzn:PermanentResidentCard",
+    "Person": "ctzn:Person",
+
+    "birthCountry": "schema:birthCountry",
+    "birthDate": {"@id": "schema:birthDate", "@type": "xsd:dateTime"},
+    "familyName": "schema:familyName",
+    "gender": "schema:gender",
+    "givenName": "schema:givenName",
+    "image": "schema:image",
```

Comment:
```suggestion
    "image": {"@id": "schema:image", "@type": "@id"},
```

Reply:
@davidlehn we're using this field for base64 encoded images, is this compatible?

Reply:
Yes, assuming you are using `data:` URIs.